platform_umbrella/apps/common_core/lib/common_core/defaults/images.ex

```diff
@@ -3,316 +3,13 @@ defmodule CommonCore.Defaults.Images do
-      Image.new!(%{
-        name: "ghcr.io/aquasecurity/trivy-operator",
-        tags: ~w(0.22.0),
-        default_tag: "0.22.0"
-      }),
-    aqua_node_collector:
-      Image.new!(%{
-        name: "ghcr.io/aquasecurity/node-collector",
-        tags: ~w(0.3.1),
-        default_tag: "0.3.1"
-      }),
-    aqua_trivy_checks:
-      Image.new!(%{
-        name: "ghcr.io/aquasecurity/trivy-checks",
-        tags: ~w(0.13.0),
-        default_tag: "0.13.0"
-      }),
-    trust_manager:
-      Image.new!(%{
-        name: "quay.io/jetstack/trust-manager",
-        tags: ~w(v0.11.0 v0.12.0),
-        default_tag: "v0.11.0"
-      }),
-    trust_manager_init:
-      Image.new!(%{
-        name: "quay.io/jetstack/cert-manager-package-debian",
-        tags: ~w(20210119.0),
-        default_tag: "20210119.0"
-      }),
-    vm_operator:
-      Image.new!(%{
-        name: "victoriametrics/operator",
-        tags: ~w(v0.44.0),
-        default_tag: "v0.44.0"
-      }),
-    knative_serving_activator:
-      Image.new!(%{
-        name: "gcr.io/knative-releases/knative.dev/serving/cmd/activator",
-        tags: @knative_allowed_tags,
-        default_tag: @knative_default_tag
-      }),
-    knative_serving_autoscaler:
-      Image.new!(%{
-        name: "gcr.io/knative-releases/knative.dev/serving/cmd/autoscaler",
-        tags: @knative_allowed_tags,
-        default_tag: @knative_default_tag
-      }),
-    knative_serving_controller:
-      Image.new!(%{
-        name: "gcr.io/knative-releases/knative.dev/serving/cmd/controller",
-        tags: @knative_allowed_tags,
-        default_tag: @knative_default_tag
-      }),
-    knative_serving_queue:
-      Image.new!(%{
-        name: "gcr.io/knative-releases/knative.dev/serving/cmd/queue",
-        tags: @knative_allowed_tags,
-        default_tag: @knative_default_tag
-      }),
-    knative_serving_webhook:
-      Image.new!(%{
-        name: "gcr.io/knative-releases/knative.dev/serving/cmd/webhook",
-        tags: @knative_allowed_tags,
-        default_tag: @knative_default_tag
-      }),
-    knative_istio_controller:
-      Image.new!(%{
-        name: "gcr.io/knative-releases/knative.dev/net-istio/cmd/controller",
-        tags: @knative_istio_allowed_tags,
-        default_tag: @knative_istio_default_tag
-      }),
-    knative_istio_webhook:
-      Image.new!(%{
-        name: "gcr.io/knative-releases/knative.dev/net-istio/cmd/webhook",
-        tags: @knative_istio_allowed_tags,
-        default_tag: @knative_istio_default_tag
-      }),
-    __schema_test:
-      Image.new!(%{
-        name: "ecto/schema/test",
-        tags: ~w(1.2.3 1.2.4 latest),
-        default_tag: "1.2.3"
-      })
-  }
+  @batteries_included_base "#{CommonCore.Version.version()}-#{CommonCore.Version.hash()}"
 
   @spec get_image(atom()) :: Image.t() | nil
   def get_image(name) do
-    @registry
-    |> Enum.filter(fn {k, _v} -> k == name end)
-    |> Enum.map(&elem(&1, 1))
-    |> List.first()
```

Comment:
I'm not sure why this used a pipeline. I think it just didn't evolve with the registry? Or am I missing something?